tests/testsuite/freshness.rs

```diff
@@ -2799,7 +2799,6 @@ fn lld_is_fresh() {
             r#"
                 [target.x86_64-pc-windows-msvc]
                 linker = "rust-lld"
-                rustflags = ["-C", "link-arg=-fuse-ld=lld"]
```

Comment:
Thanks! Could you also help remove this as well?

https://github.com/rust-lang/cargo/blob/91d8140d66e86127348c1230c9b913effca3d1c9/tests/testsuite/freshness_checksum.rs?plain=1#L2605

Reply:
sure thing, done :)